package.json

```diff
@@ -6,7 +6,9 @@
   "jsnext:main": "esm/index.js",
   "typings": "index.d.ts",
   "dependencies": {
-    "@angular/core": ">= 2.0.0-rc.5"
+    "@angular/core": ">= 2.0.0",
+    "rxjs": "5.0.0-beta.12",
```

Comment:
I know that @angular/core has rxjs and zone.js as peer dependencies but I can't start the project without them with simple `npm install`. That's why I added them, but I revert that now.